pkg/cvo/cvo.go

```diff
@@ -223,7 +223,15 @@ func New(
 // controller that loads and applies content to the cluster. It returns an error if the payload appears to
 // be in error rather than continuing.
 func (optr *Operator) InitializeFromPayload(restConfig *rest.Config, burstRestConfig *rest.Config) error {
-	update, err := payload.LoadUpdate(optr.defaultPayloadDir(), optr.release.Image, optr.exclude, optr.clusterProfile)
+	includeTechPreview := func() (bool, error) {
+		gate, err := optr.client.ConfigV1().FeatureGates().Get(context.TODO(), "cluster", metav1.GetOptions{})
```

Comment:
We'll want to have an actual `Context` input argument to this function before we land this, not the `TODO`, although obviously whatever's easiest for testing while you're still sketching.

Reply:
> We'll want to have an actual `Context` input argument to this function before we land this, not the `TODO`, although obviously whatever's easiest for testing while you're still sketching.

this has since been corrected and moved.